cmd/crio/main.go

```diff
@@ -204,6 +204,46 @@ func mergeConfig(config *libconfig.Config, ctx *cli.Context) (string, error) {
 	if ctx.GlobalIsSet("additional-devices") {
 		config.AdditionalDevices = ctx.GlobalStringSlice("additional-devices")
 	}
+	if ctx.GlobalIsSet("conmon-env") {
+		config.ConmonEnv = ctx.GlobalStringSlice("conmon-env")
+	}
+	if ctx.GlobalIsSet("container-attach-socket-dir") {
+		config.ContainerAttachSocketDir = ctx.GlobalString("container-attach-socket-dir")
+	}
+	if ctx.GlobalIsSet("container-exits-dir") {
+		config.ContainerExitsDir = ctx.GlobalString("container-exits-dir")
+	}
+	if ctx.GlobalIsSet("ctr-stop-timeout") {
+		config.CtrStopTimeout = ctx.GlobalInt64("ctr-stop-timeout")
```

Comment:
Why is this ctr and the others are container?

Reply:
The flags are just converted from the `internal/lib/config/config.go` tags to have a consistent naming between both:
https://github.com/cri-o/cri-o/blob/395ca4951cb953bf2559232118c108d491c614ce/internal/lib/config/config.go#L243-L245